mbed_greentea/mbed_greentea_cli.py

```diff
@@ -141,7 +141,8 @@ def create_filtered_test_list(ctest_test_list, test_by_names, skip_test, test_sp
         gt_logger.gt_log_tab("note: see list of available test cases below")
         # Print available test suite names (binary names user can use with -n
         if test_spec:
-            list_binaries_for_builds(test_spec)
+            parsed_test_spec = TestSpec(test_spec)
```

Comment:
This is fixed here: https://github.com/ARMmbed/greentea/pull/130/commits/e274f0299b65ebf98269fa237f225c05ed1a5150,